examples/SpatialDetection/spatial_location_calculator.py

```diff
@@ -2,6 +2,7 @@
 
 import cv2
 import depthai as dai
+from utility import getDisparityScaleFactor, colorizeDepthPacket
```

Comment:
The code doesn't work because utility is not in the same folder.
Fails at import.
Same applies to all examples that has been changed.